nim/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nim",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://ai.nim.network",
+  "pretty_name": "Nim Network",
+  "chain_id": "nim_1122-1",
+  "bech32_prefix": "nim",
+  "slip44": 60,
+  "node_home": "",
+  "daemon_name": "",
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "anim",
+        "low_gas_price": 20000000000,
+        "average_gas_price": 20000000000,
+        "high_gas_price": 20000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "anim"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nim/images/nim.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nim/images/nim.svg"
+  },
+  "description": "Nim Network is a highly-adoptable AI Gaming chain that will provide the ultimate ecosystem for exploration and development of games at the intersection of Web3 and AI.",
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nim/images/dymension-logo.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nim/images/dymension-logo.svg"
+    }
+  ],
+  "codebase": {
+    "git_repo": "",
+    "recommended_version": "v2.1.3-rc02",
+    "compatible_versions": [
+      "v2.1.3-rc02"
+    ],
+    "genesis": {
+      "genesis_url": ""
+    }
+  },
+  "peers": "",
```

Comment:
peers in an array, not a string, but let's just omit properties with blank/empty values